h3i/src/recordreplay/qlog.rs

```diff
@@ -336,6 +369,33 @@ impl From<&PacketSent> for H3Actions {
                         error_code: *error_code,
                     }),
 
+                    QuicFrame::ConnectionClose {
+                        error_space,
+                        error_code,
+                        reason,
+                        ..
+                    } => {
+                        let is_app = error_space
```

Comment:
qlog allows any of these fields to be `None`, so the code should be more defensive. If there is no error_space or error_code, then there is no reasonable default to assume and we should log it and either bomb out, or skip the action.

Reply:
Changed to `expect` the error space and added a more descriptive `expect` on the error code, rather than unwrap